lighty-core/lighty-controller/pom.xml

```diff
@@ -493,6 +522,10 @@
             <artifactId>singlenode-configuration</artifactId>
             <scope>test</scope>
         </dependency>
+
+
+
+
```

Comment:
do not introduce unnecessary empty lines

Reply:
oh no